cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -393,8 +393,8 @@ func (sub EthereumSub) logToEvent(networkDescriptor oracletypes.NetworkDescripto
 		event.ClaimType = ethbridgetypes.ClaimType_CLAIM_TYPE_BURN
 		event.To = append(event.To, decodedEvent.To...)
 		// burn doesn't have symbol or name
-		//event.Symbol = decodedEvent.Symbol
-		//event.Name = decodedEvent.Name
+		event.Symbol = decodedEvent.Denom
```

Comment:
but we need the denom from burn event, for the token like ibc and rowan, the denom not got from network descriptor and smart contract. just put it in symbol field, since there is not denom defined.

Reply:
Doesn't build for me:

```
cmd/sifnoded/cmd/root.go:85:3: undefined: SetTokenRegistryAdminCmd
make: *** [Makefile:49: install] Error 2
```